applications/ecallmgr/src/ecallmgr_call_control.erl

```diff
@@ -360,9 +361,9 @@ handle_call_events(JObj, FetchId) ->
     kz_util:put_callid(kz_json:get_value(<<"Call-ID">>, JObj)),
     case kz_api:event_name(JObj) of
         <<"usurp_control">> ->
-            case kz_json:get_ne_binary_value(<<"Fetch-ID">>, JObj) =:= FetchId of
-                'false' -> gen_listener:cast(self(), {'usurp_control', JObj});
-                'true' -> 'ok'
+            case kz_json:get_ne_binary_value(<<"Fetch-ID">>, JObj) =/= FetchId of
```

Comment:
@jamesaimonetti i don't understand this change

Reply:
I kept reading it as "if JObj's FetchId matches our FetchId then usurp control". It made me go down a wrong path trying to usurp the loopback's call control process and I was really mad. I found this to be easier to parse into English in my head :)